docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -1,42 +1,91 @@
-////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
-https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
-////
-= Quarkus - Quarkus Extension for Spring DI API
 
-include::./attributes.adoc[]
-:extension-status: preview
 
-While users are encouraged to use CDI annotations for injection, Quarkus provides a compatibility layer for Spring dependency injection in the form of the `spring-di` extension.
+// comment out to show community version
+:product-docs:
+// TODO: consider moving this to the attributes file
+:doctype: book
 
-This guide explains how a Quarkus application can leverage the well known Dependency Injection annotations included in the Spring Framework.
+//TODO: Determine if title can be replaced by attribute
+//Top-Level Assembly
+[id="using-the-quarkus-extension-for-spring-di-api_{context}"]
+= Using the Quarkus extension for Spring Dependency Injection (DI) API
 
-include::./status-include.adoc[]
+ifdef::context[:parent-context: {context}]
 
-== Prerequisites
+:context: using-the-quarkus-extension-for-spring-di
 
-To complete this guide, you need:
+include::./attributes.adoc[]
 
-* less than 15 minutes
-* an IDE
-* JDK 1.8+ installed with `JAVA_HOME` configured appropriately
-* Apache Maven {maven-version}
+[role="_abstract"]
+The `quarkus-spring-di` extension provides a compatibility layer for Spring Dependency Injection that you can use as an alternative to MicroProfile CDI annotations in your {ProductName} application.
+This guide explains how your {ProductName} application can leverage Dependency Injection annotations provided by the Spring Framework.
 
+ifndef::product-docs[]
+
+This guide is maintained in the main {ProductName} repository and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
 
 == Solution
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
-However, you can go right to the completed example.
+Follow the instructions in the next sections and create the application step by step.
+Alternatively, you can clone and use the completed example.
 
 Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
 
 The solution is located in the `spring-di-quickstart` {quickstarts-tree-url}/spring-di-quickstart[directory].
+endif::[]
 
-== Creating the Maven project
-
-First, we need a new project. Create a new project with the following command:
-
+// Tech preview info for community purposes, different from product
+ifndef::product-docs[]
+include::./status-include.adoc[]
```

Comment:
Why are this removed. This status is not related to red hat support ability.

Reply:
I added the conditional to make the note visible in the community version only, because we need to provide a note about the preview status of this extension that is specific to RHBQ in product docs.